include/alpaca/detail/to_bytes.h

```diff
@@ -73,6 +73,55 @@ to_bytes(T &bytes, std::size_t &byte_index, const U &original_value) {
   }
 }
 
+
+
+// encode size_t as fixed 4-bytes integer
+template <options O, typename T, typename U>
+typename std::enable_if<
+        std::is_same_v<U, std::size_t>,
```

Comment:
Should this be just `size_t` instead of `std::size_t`? To be consistent with the rest of your code in `from_bytes.h`